pkg/antctl/raw/multicluster/common/common.go

```diff
@@ -303,6 +309,86 @@ func CreateMemberToken(cmd *cobra.Command, k8sClient client.Client, name string,
 	return nil
 }
 
+func DeleteMemberToken(cmd *cobra.Command, k8sClient client.Client, name string, namespace string) error {
+	errFunc := func(kind string, act string, err error) error {
+		if apierrors.IsNotFound(err) {
+			fmt.Fprintf(cmd.OutOrStdout(), "%s %s not found in Namespace %s\n", kind, name, namespace)
+			return nil
+		}
+		if err != nil {
+			fmt.Fprintf(cmd.OutOrStderr(), "Failed to %s %s \"%s\": %v\n", act, kind, name, err)
+			return err
+		}
+		if act == "get" {
+			return nil
+		} else {
+			fmt.Fprintf(cmd.OutOrStdout(), "%s %s deleted successfully\n", kind, name)
+			return nil
+		}
+	}
+
+	secret := &corev1.Secret{}
+	getErr := k8sClient.Get(context.TODO(), types.NamespacedName{Namespace: namespace, Name: name}, secret)
+	err := errFunc("Secret", "get", getErr)
+	if err != nil {
+		return err
+	}
+	if secret.Annotations[CreateByAntctlAnnotation] == "true" {
```

Comment:
Is it safe to access Annotations, when not found?

Reply:
@jianjuns I think this is safe, we have a unit test for it.